tests/api/test_collect_runs_logs.py

```diff
@@ -182,6 +184,63 @@ async def test_start_log_for_run_failure(
         assert run_uid is None
         assert log_collector._call.call_count == 1
 
+    @pytest.mark.asyncio
+    async def test_log_collector_sidecar(
+        self, db: sqlalchemy.orm.session.Session, client: fastapi.testclient.TestClient
+    ):
+        # TODO: start log collector sidecar in test
+
+        # set config log collector sidecar values to enable it
+        prev_log_collector_mode = mlrun.config.config.log_collector.mode
+        prev_log_collector_address = mlrun.config.config.log_collector.address
```

Comment:
we are using `config_test_base` in `common_fixtures.py` which is in autouse to clear configuration between tests, therefore I would move it to there